bin/main.ml

```diff
@@ -749,6 +782,47 @@ let resolve_targets_exn ~log common setup user_targets =
     | Ok targets ->
       targets)
 
+let fswatch_command root_path =
+  let excludes = [ {|/\.#|}
```

Comment:
A few elements in this list only seem relevant when not doing out-of-source builds. Let's restrict it to:
```
                  [ {|\.#|}
                  ; {|_build|}
￼                  ; {|\.hg|}
￼                  ; {|\.git|}
￼                  ; {|~$|}
￼                  ; {|/#[^#]*#$|}
￼                  ; {|\.install$|}
                  ]
```